projects/plugins/jetpack/_inc/client/traffic/related-posts.jsx

```diff
@@ -101,111 +259,26 @@ class RelatedPostsComponent extends React.Component {
 						toggleModule={ this.props.toggleModuleNow }
 					>
 						<span className="jp-form-toggle-explanation">
-							{ __( 'Show related content after posts', 'jetpack' ) }
+							{ __( 'Show related content for each one of your posts.', 'jetpack' ) }
```

Comment:
Might it make sense to note that this feature doesn't work for block themes as part of this explanation?

Reply:
The general toggle still applies to block themes, so I think it's okay there. What doesn't work with block themes is hidden from the UI when you use such a theme.